tree/ntuple/v7/test/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if(MSVC)
                                      ${CMAKE_CURRENT_BINARY_DIR}/libCustomStruct.dll)
 endif()
 ROOT_ADD_GTEST(ntuple ntuple.cxx LIBRARIES ROOTDataFrame ROOTNTuple MathCore CustomStruct)
+ROOT_ADD_GTEST(RNtuple RNtuple.cxx LIBRARIES ROOTDataFrame ROOTNTuple MathCore CustomStruct)
```

Comment:
Please rename file in ntuple_compression.cxx, the same about name of executable.
Do you really need CustomStruct library as dependency?